manifests/params.pp

```diff
@@ -91,6 +91,7 @@
   $conf_d                     = "${confdir}/conf.d"
   $config_file                = "${confdir}/unbound.conf"
   $control_enable             = false
+  $control_setup_path         = '/usr/sbin/unbound-control-setup'
```

Comment:
i'm pretty and sure that this will be different on different platforms…

maybe we should just use PATH in the exec?